controls/pcidss_ocp4.yml

```diff
@@ -144,68 +181,95 @@ controls:
+      source address.)
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      Anti-spoofing and blocking forged IP addresses is outside the scope
+      of the OpenShift Container Platform.
+    status: not applicable
     rules: []
 
   - id: Req-1.3.4
-    title: 1.3.4 Implement anti-spoofing measures to detect and block forged source
-      IP addresses from entering the network.
-    description: (For example, block traffic originating from the Internet with an internal
-      source address.)
+    title: >-
+      1.3.4 Do not allow unauthorized outbound traffic from the cardholder
+      data environment to the Internet.
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      OpenShift can support authorization of outbound traffic via
+      egress  policy  objects[1][2].
+
+      [1] https://docs.openshift.com/container-platform/4.6/networking/openshift_sdn/configuring-egress-firewall.html
+      [2] https://docs.openshift.com/container-platform/4.6/networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.html
+    status: supported
     rules: []
 
   - id: Req-1.3.5
-    title: 1.3.5 Do not allow unauthorized outbound traffic from the cardholder data
-      environment to the Internet.
+    title: 1.3.5 Permit only "established" connections into the network.
     levels:
     - base
     notes: ''
     status: pending
     rules: []
 
   - id: Req-1.3.6
-    title: "1.3.6 Implement stateful inspection, also known as dynamic packet filtering.\
-      \ (That is, only \u201Cestablished\u201D connections are allowed into the network.)"
+    title: >-
+      1.3.6 Place system components that store cardholder data (such as a
+      database) in an internal network zone, segregated from the DMZ and other
+      untrusted networks
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    notes: |-
+      Segregation of such system components can take place using the same
+      network segregation mechanisms that OpenShift already supports [1][2][3]:
 
-  - id: Req-1.3.7
-    title: 1.3.7 Place system components that store cardholder data (such as a database)
-      in an internal network zone, segregated from the DMZ and other untrusted networks.
-    levels:
-    - base
-    notes: ''
-    status: pending
-    rules: []
+      * Ingress
+      * NetworkPolicies
+      * Egress Policies
+
+      Special attention needs to be taken to audit that these components
+      are appropriately segregated by the aforementioned mechanisms
 
-  - id: Req-1.3.8
-    title: 1.3.8 Do not disclose private IP addresses and routing information to unauthorized
-      parties.
+      [1] https://docs.openshift.com/container-platform/4.8/networking/network_policy/about-network-policy.html
+      [2] https://docs.openshift.com/container-platform/4.6/networking/openshift_sdn/configuring-egress-firewall.html
```

Comment:
Should we point to 4.8 docs for the rest of these? (or use the /latest path)

Reply:
done